code/controllers/subsystems/job.dm

```diff
@@ -393,7 +393,7 @@
 	to_chat(H, "<B>You are [job.total_positions == 1 ? "the" : "a"] [alt_title ? alt_title : rank].</B>")
 
 	if(job.supervisors)
-		to_chat(H, "<b>As the [alt_title ? alt_title : rank] you answer directly to [job.supervisors]. Special circumstances may change this.</b>")
+		to_chat(H, "<b>As [job.intro_prefix] [alt_title ? alt_title : rank] you answer directly to [job.supervisors]. Special circumstances may change this.</b>")
```

Comment:
This seems like a waste of a var tbh. Isn't the following a valid construct in English: "As Medical Doctor/Visitor/rank you answer to..."?

Reply:
I thought about it, and yeah, technically both are correct in some contexts.  Without the connecting word though you are referring to Medical Doctor and Captain as a proper noun/title, not a job.  While that works in some cases, it's more awkward in relaxed designations like 'visitor' and 'assistant', because those aren't titles, they're rankings or positions.

I think it's much smoother to treat the title as a designation instead of a proper noun.  You can also do things like use 'as the Captain' to indicate by subtle context that there is only one of them, whereas 'a medical doctor' indicates that it is not a singular position.